### PR TITLE
Add support for libswiftnav L2C signal updates

### PR DESCRIPTION
**Pull request by pmiettinen**
https://github.com/swift-nav/exafore_planning/issues/36

### CODE REVIEW

**Comment by pmiettinen** [comment]
Please note python/tests/sbp/utils.py change! Encoded message bytes needs to be updated in order to make the tests valid again.

**Comment by pmiettinen** [comment]
https://github.com/swift-nav/exafore_planning/issues/36

**Comment by mfine** [comment]
@JoshuaGross should we be checking in all the auto generated test stuff for C?

File: spec/yaml/swiftnav/sbp/gnss_signal.yaml [hunk 1]
```diff
@@ -24,9 +24,9 @@ definitions:
...
-        - band:
+        - code:
             type: u8
-            desc: Signal band
-        - constellation:
+            desc: Signal constellation, band and code
+        - reserved:
             type: u8
-            desc: Constellation to which the satellite belongs
+            desc: Reserved
```
Comment (mfine) [inline spec/yaml/swiftnav/sbp/gnss_signal.yaml]: @gsmcmullin @jacobmcnamee @denniszollo @fnoble no binary changes here, but there is a renaming here: band -> code, constellation -> reserved. Just want to make sure everyone's ok on this.

**Comment by jacobmcnamee** [comment]
lgtm.

Probably personal preference, but it would be nice if we could separate the real changes from the auto-generated stuff. Makes it much easier to review.

**Pull request merged**